### DISCUSSION TITLE
支持 Firefox 浏览器讨论

### DISCUSSION THREAD

**Discussion by starknt** [BewlyBewly/BewlyBewly#110] [General]
当前在 #109 中我已完成了一些支持 Firefox 浏览器的工作，但还有一些问题需要解决

**Comment by hakadao** [comment]
那邊firefox有時載入不了啊，頁面都沒有出現
<img width="1920" alt="image" src="https://github.com/hakadao/BewlyBewly/assets/33394391/e66a6d89-435d-4c04-8ace-6f17a687114b">

**Comment by hakadao** [comment]
現在動向頁就是這樣，不知道怎麼無緣無故説跨域問題然後頁面又不能載入
看來也只能向別人求助了。。。。。
<img width="1920" alt="image" src="https://github.com/hakadao/BewlyBewly/assets/33394391/574805e2-cfcb-4161-b1aa-cf036e5719c9">

**Comment by starknt** [comment]
对 Firefox 浏览器的兼容的主要问题已在 #113 完全解决

**Comment by hakadao** [comment]
正式支持目前在 dev 分支中, 終於解決了
